src/main/java/step3_renewal/ladderApplication.java

```diff
@@ -0,0 +1,10 @@
+package step3_renewal;
+
+
+import step3_renewal.controller.LadderGameController;
+
+public class ladderApplication {
```

Comment:
```suggestion
public class LadderApplication {
```
클래스 이름은 대문자로 시작하도록 해주세요! 😄